src/renderers/shared/shared/event/BrowserEventConstants.js

```diff
@@ -64,6 +64,8 @@ var topLevelTypes = {
   topLoadedMetadata: 'loadedmetadata',
   topLoadStart: 'loadstart',
   topMouseDown: 'mousedown',
+  topMouseEnter: 'mouseenter',
+  topMouseLeave: 'mouseleave',
```

Comment:
`topLevelTypes` is imported in `ReactBrowserEventEmitter`, see: https://github.com/facebook/react/blob/master/src/renderers/dom/shared/ReactBrowserEventEmitter.js#L19